### PR TITLE
add(gitignore): added gradle and .build file to gitignore

### PR DESCRIPTION
**Pull request by neubs-bsi**
Added all gradle build folders to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,6 @@
 *.iml
 .gitreview
 velocity.log
-antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
\ No newline at end of file
+antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
+**/.gradle
```
Comment (maxhbr) [inline .gitignore]: `**/.gradle` is equivalent to simply writing `.gradle`

**Comment by neubs-bsi** [comment]
@maxhbr updated it to your review.

File: .gitignore [hunk 2]
```diff
@@ -8,4 +8,6 @@
 *.iml
 .gitreview
 velocity.log
-antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
\ No newline at end of file
+antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
+.gradle
+**/build
```
Comment (maxhbr) [inline .gitignore]: sorry, same issue here, leading `**/` is always unnecessary.

Comment (neubs-bsi) [reply .gitignore]: changed it once more :)

**Pull request merged**